x-pack/plugins/gis/public/shared/layers/layer.js

```diff
@@ -39,7 +39,7 @@ export class ALayer {
     layerDescriptor.minZoom = _.get(options, 'minZoom', 0);
     layerDescriptor.maxZoom = _.get(options, 'maxZoom', 24);
     layerDescriptor.alpha = _.get(options, 'alpha', 0.75);
-    layerDescriptor.visible = options.visible || true;
+    layerDescriptor.visible = _.get(options, 'visible', true);
     layerDescriptor.temporary = options.temporary || false;
```

Comment:
Looks like the next line (temporary) will have the same problem. How about updating temporary and style as well just to use lodash to avoid future problems